code/modules/antagonists/traitor/equipment/contractor.dm

```diff
@@ -0,0 +1,171 @@
+GLOBAL_LIST_INIT(contractor_items, subtypesof(/datum/contractor_item))
```

Comment:
This seems bit redundant, but i suppose saves repeated subtypes. Probably better off as static.